tools/serverpod_cli/test/integration/util/model_helper_test.dart

```diff
@@ -217,7 +217,7 @@ void main() {
       expect(
         paths.first,
         contains(
-            'test/integration/util/test_assets/protocol_helper/has_serverpod_server_project/test_server/lib/src/model/example.spy.yaml'),
+            'test/integration/util/test_assets/protocol_helper/has_serverpod_server_project/test_server/lib/src/models/example.spy.yaml'),
```

Comment:
Note: I think this test will fail if we ever run it on windows 🙃

Reply:
I believe we are running them on windows actually